src/chunk.rs

```diff
@@ -0,0 +1,73 @@
+use ::image::{ImageBuffer, Rgba, RgbaImage};
```

Comment:
Since this file is part of the paint canvas, I think it should belong to the `paint_canvas` module rather than be its own thing. (move this file from `src/chunk.rs` to `src/paint_canvas/chunk.rs` and add `mod chunk;` to `src/paint_canvas.rs` - I know this will be inconsistent with our current style of using `mod.rs` files but we'll get to restructuring the codebase to use the new style some other day)